### PR TITLE
no edX support for graphical slider

### PR DESCRIPTION
**Pull request by lamagnifica**
@explorerleslie , @mhoeber this fixes https://openedx.atlassian.net/browse/DOC-2291

### CODE REVIEW

File: en_us/olx/source/components/graphical_slider_tool.rst [hunk 1]
```diff
@@ -4,6 +4,8 @@
 Graphical Slider Tool
 *********************************************
 
+.. note:: EdX does not support this tool.
+
 .. module:: xml_format_gst
```
Comment (mhoeber) [inline en_us/olx/source/components/graphical_slider_tool.rst]: line 9 can be removed

**Comment by mhoeber** [comment]
:+1:  thanks.  also, do we want this in table here:  http://edx.readthedocs.org/projects/edx-partner-course-staff/en/latest/exercises_tools/create_exercises_and_tools.html

**Comment by lamagnifica** [comment]
well, @mhoeber that's an interesting question. All of the other tools in that table are also described in that guide. If we add this one to the table, shouldn't we also add it to the guide? I really don't understand how/why it ended up in the OLX guide only, and really it seems like we should be reducing the amount of attention paid to it rather than increasing its potential audience... Let me know if you want me to add it anywhere else, though.

**Comment by mhoeber** [comment]
@lamagnifica  ok, makes sense to leave as is.

**Pull request merged**